changelog.md

```diff
@@ -4,7 +4,8 @@
 * None.
 
 ### New Features
-* Updated to evolve-grpc 0.26.0.
+* Updated to evolve-grpc 0.26.0-SNAPSHOT3
```

Comment:
FYI mention of the snapshot is not needed - the changelog only gets "released" with each release, so the comments within the changelog should pertain to the specific future release that it's relevant for.

e.g everything under 0.17.0 will encapsulate all changes made in all the snapshots leading up to 0.17.0.
We know that evolve-grpc will be released prior to this, so it'll just be the released version that matters (0.26.0).